internal/datastore/mention.go

```diff
@@ -0,0 +1,40 @@
+package datastore
+
+import (
+	"context"
+	"database/sql"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/nedrocks/delphisbe/graph/model"
+	"github.com/sirupsen/logrus"
+)
+
+func (d *delphisDB) PutActivity(ctx context.Context, tx *sql.Tx, post *model.Post) error {
+	logrus.Infof("PutActivity::SQL Create")
+	if err := d.initializeStatements(ctx); err != nil {
+		logrus.WithError(err).Error("PutActivity::failed to initialize statements")
+		return err
+	}
+
+	for _, entityID := range post.PostContent.MentionedEntities {
+		s := strings.Split(entityID, ":")
```

Comment:
Can we extract this (and the other place) into a utility function? I worry about panics here if the string is malformed.

Reply:
split and check for two tokens; return an object